src/Command/DispatchCommand.php

```diff
@@ -688,6 +694,8 @@ private function renderFile(Package $package, string $repositoryName, string $br
                     'package_title' => ucwords(str_replace(['-project', '/', '-'], ['', ' ', ' '], $package->getName())),
                     'package_description' => $package->getDescription(),
                     'packagist_name' => $package->getName(),
+                    'packagist_homepage' => $this->getLastPackageVersion($package)->getHomepage(),
+                    'packagist_keywords' => $this->getLastPackageVersion($package)->getKeywords(),
```

Comment:
Do we really need them for the templates? 🤔

Reply:
No indeed